src/style.css

```
@@ -0,0 +1,44 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  background-position: center;
+  align-items: center;
+  justify-content: center;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  margin: 0 2px;
+  background-image: url(images/star.svg);
+  background-position: top 2px left 0;
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars_star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--2 .stars_star:nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--3 .stars_star:nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--4 .stars_star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--5 .stars_star:nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}

```

```suggestion
.stars--1 .stars_star:nth-child(1),
.stars--2 .stars_star:nth-child(-n+2),
.stars--3 .stars_star:nth-child(-n+3),
.stars--4 .stars_star:nth-child(-n+4),
.stars--5 .stars_star:nth-child(-n+5) {
  background-image: url(images/star-active.svg);
}
```
Це все можна об'єднати
background-repeat: no-repeat тут не варто вказувати, адже це вже прописано в класі stars_star